src/types/MovieCardProps.ts

```diff
@@ -0,0 +1,5 @@
+import { Movie } from './Movie';
+
+export interface MovieCardProps {
```

Comment:
I'm not sure if it is important, but Misha said that for props it's better to use type instead of interface and interface for data that we will receive from the server

Reply:
It's not) Interface is a good solution
Also, you can find info about different about Type & Interface